curandommatrices/src/CuRandomMatrices.jl

```diff
@@ -0,0 +1,15 @@
+module CuRandomMatrices
+export curand
```

Comment:
`curand()` methods should allow for setting the type of the returned value. For example it is possible to call `CuArrays.rand(Float64, 3, 3)`.

Reply:
This can be implemented when sampling on the CPU is reworked and `rand` supports this. The question is whether this should be supported by `rand`/`curand` or the types themselves, something like
```
struct GinibreEnsemble{\beta, T}
...
```
Both cases need to keep track which types are appropriate for every distribuiton.